### DISCUSSION TITLE
Étude de cas : "Ethnographie"

### DISCUSSION THREAD

**Discussion by RomainUTT** [Hypertopic/HyperGlosae#135] [Show and tell]
Here is the video presenting all the content we added to Hyperglosae.

https://github.com/Hypertopic/HyperGlosae/assets/126058629/dd69cae6-0543-4539-aaa2-782a650ce563

**Comment by benel** [comment]
Merci à toute l'équipe pour votre contribution à la collaboration avec M. Lejeune et au logiciel lui-même.

Je laisse la parole à @christophe-lejeune pour des commentaires ou des questions.

**Comment by philacour** [comment]
Pertinent, bravo

**Comment by christophe-lejeune** [comment]
Votre travail met en évidence trois usages potentiels. Deux sont relativement explicites, et font partie de ce dont nous avions discuté. Un troisième m'est apparu en visionnant cette vidéo. 

* Le premier usage correspond finalement à l'usage "générique" des types, indépendamment du contexte. Il s'agit ici de typer des documents, en mobilisant soit un nouveau type (créé par l'utilisateur) soit un type stocké dans la base de données.
* Le deuxième usage correspond à ce que propose [Cassandre](/Hypertopic/Cassandre). Il correspond à l'utilisation de types (stockés dans la base de données) qui correspondent à une typologie préexistante.
* Le troisième correspond en fait à une déclinaison du deuxième. Dans [Cassandre](/Hypertopic/Cassandre), le jeu de types est inscrit "en dur" dans l'infrastructure et le code du logiciel (comme vous l'aviez fait dans un premier temps). En déplaçant les types du code vers la base de données, vous ouvrez la possibilité de disposer de différents jeux de types. Ainsi, sans amender profondément le logiciel, on pourrait l'utiliser pour accompagner d'autres méthodes qualitatives, comme l'analyse phénoménologique interprétative (IPA), par exemple, qui dispose de trois types de notes, correspondant à différents "paliers" d'analyse (ce qui diffère de l'analyse par théorisation ancrée). Évidemment, l'IPA est un exemple parmi d'autres. Il serait assez facile de disposer de plusieurs jeux correspondant aux différentes méthodes, ce qui rend également le logiciel très "accueillant" par rapport à d'éventuelles nouvelles formalisations méthodologiques.

Je vois deux prolongations possibles à ce qui a été développée jusqu'ici.
 
* Tout d'abord, une question d'interface. Je me demande si les couleurs associées aux types (et apparaissant comme une étiquette sur le document) ne pourraient pas être répercutées sur l'icône apparaissant à gauche du nom du document (cette icône reste actuellement rouge quoi qu'il arrive). Dans le même esprit, il pourrait être pratique d'associer une icône à chaque type. Ainsi l’icône en question pourrait se substituer à l'icône rouge actuelle. Cette proposition correspond en fait à ce que [Cassandre](/Hypertopic/Cassandre) fait actuellement. Je vous concède qu'il s'agit essentiellement de cosmétique... mais peut-être l'interface en serait-elle gagnerait-elle en légèreté et en "affordance". 
* Pour améliorer le gain des différents jeux de types, il me semblerait utile de pouvoir structurer/scripter la succession des types de documents, comme cela a été discuté dans [un autre ticket](https://github.com/Hypertopic/HyperGlosae/issues/113) 

En tout cas, merci pour le travail accompli et excellente continuation à chaque membre de l'équipe !
